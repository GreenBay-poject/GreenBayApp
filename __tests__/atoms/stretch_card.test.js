import React from 'react';
import renderer from 'react-test-renderer';

import StretchCard from '../../atoms/stretch_card'

describe('<StretchCard />', () => {
    it('Sample StretchCard', () => {
        const tree = renderer.create(<StretchCard imagelink={"book"} paragraph={"Sample Paragraph"} title={"Sample"} navigation={null} page_to_go={"Dashboard"} />).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.children[0].children[0].type).toBe('View');
        expect(tree.children[0].children[1].type).toBe('View');
        //console.log(tree.children[0].children[0].type)

    });
});