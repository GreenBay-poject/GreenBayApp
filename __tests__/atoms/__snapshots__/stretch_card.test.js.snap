// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StretchCard /> Sample StretchCard 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "stretch",
      "borderColor": "#153d37",
      "borderRadius": 30,
      "borderWidth": 1,
      "elevation": 4,
      "marginHorizontal": 20,
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#d7dbd8",
        "borderRadius": 30,
        "flex": 1,
        "flexDirection": "row",
        "height": 120,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#d7dbd8",
          "borderRadius": 30,
          "flex": 1.5,
          "fontSize": 17,
          "fontWeight": "bold",
          "paddingTop": 20,
        }
      }
    >
      <Text />
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#d7dbd8",
          "borderRadius": 30,
          "flex": 3,
          "fontSize": 15,
          "textAlign": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#153d37",
            "fontSize": 17,
            "fontWeight": "bold",
            "marginTop": 15,
          }
        }
      >
        Sample
      </Text>
      <Text
        style={
          Object {
            "color": "#153d37",
            "fontSize": 15,
            "marginLeft": 15,
            "marginRight": 15,
            "marginTop": 15,
            "textAlign": "center",
          }
        }
      >
        Sample Paragraph
      </Text>
    </View>
  </View>
</View>
`;
